setup.py

```diff
@@ -98,9 +98,7 @@
             "Programming Language :: Python :: 3.8",
             "Topic :: Software Development",
         ],
-        test_suite="tests.AllTests",
         python_requires=">=3.5",
         install_requires=["deprecated", "pyjwt", "requests>=2.14.0"],
         extras_require={"integrations": ["cryptography"]},
-        tests_require=["cryptography", "httpretty>=0.9.6", "parameterized>=0.7.0"],
```

Comment:
I think we can probably keep the `tests_require`, as some downstream usage may depend on PyGithub for these dependencies

Reply:
Even if this this list is outdated compared to the previous test-requirements.txt? :-)

Reply:
tests_require is not the same as requirement.txt, it should provide the minimum version required
https://packaging.python.org/discussions/install-requires-vs-requirements/